pkg/core/native/native_nep17.go

```diff
@@ -174,8 +174,8 @@ func (c *nep17TokenNative) updateAccBalance(ic *interop.Context, acc util.Uint16
 		}
 		si = state.StorageItem{}
 	} else if amount.Sign() <= 0 && requiredBalance != nil {
-		// If amount == 0 then it's either a roundtrip or an empty transfer. In
-		// case of a roundtrip account's balance may still be less than actual
+		// If amount == 0 then it's either a round trip or an empty transfer. In
+		// case of a round trip account's balance may still be less than actual
```

Comment:
I don't think it strictly is an error (though I'm not against the change too): https://www.merriam-webster.com/dictionary/round%20trip